src/components/carousel/_children/IndicatorArea.jsx

```diff
@@ -0,0 +1,64 @@
+import { Children } from "react";
+
+const BLOCK_CLASS_NAME = "c-carousel";
+
+// get only the images from the nested sub components
+// recursive loop to only find the Image type component
+const getImages = (children) =>
```

Comment:
I have a concern with this approach, as the images used within the main part of the carousel will not be fully configured to be placed in multiple locations. What are we suggesting for ensuring the images used as thumbnails are sized accordingly?

Reply:
That's a good concern. I was planning on adding alternate really small sizes to the sizes array to any image that could be used in a thumbnail approach. What did you have in mind as an opportunity here?

Reply:
Will take in thumbnails as react children for the props

Reply:
addressed